index.html

```diff
@@ -21,6 +21,7 @@ <h1>Bookbinder JS</h1>
             A Javascript-based app for formatting PDFs for bookbinding. For more information, feature requests, or to
             contribute, view the project's <a href="https://github.com/momijizukamori/bookbinder-js">Github repository</a>.
             If you have issues with this version, you can try the <a href="old/">old version</a> instead, which only supports folio layouts.
+            <button type="button" name="reset_settings" id="reset_settings">Reset to Default Settings</button>
```

Comment:
I'm good with doing this! I'll also add a little info button with some help text.

Do we want to center it (may look odd with the [i]), or break it into it's own section? With a dedicated section, we would have a place if we ever wanted to do other settings-related things (file upload, for example), but it could look weird with just one button.

Reply:
Added a settings section! If we want to change the look or blurb for the information text, let me know!